EIPS/eip-1.md

```diff
@@ -39,6 +39,9 @@ An EIP must meet certain minimum criteria. It must be a clear and complete descr
 
 Parties involved in the process are you, the champion or *EIP author*, the [*EIP editors*](#eip-editors), and the [*Ethereum Core Developers*](https://github.com/ethereum/pm).
 
+### Suggestions for Authors
+*The following are not requirements but are suggestions from the EIP Editors*
```

Comment:
They are suggestions in order to get your proposal taken seriously. So it's somewhere in between requirements and suggestions.